projectDocs/dev/copyrightHeaders.md

```diff
@@ -0,0 +1,26 @@
+In NVDA files, which are licensed under the GNU GPL, use the following copyright header:
+
+```
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) <CREATED YEAR>-<LAST UPDATED YEAR> NV Access Limited, <YOUR NAME>
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
```

Comment:
Again, I thought we were operating under a modified license? If so, the files should have their headers indicate the project license, as a derivative of the GPL V2, but shouldn't say that they are licensed under GPL V2--else the modified project license has no meaning.

```suggestion
# This file may be used under the terms of the GNU General Public License, version 2 or later, as modified by the NVDA license.
```

Reply:
I can see the argument of doing it either way. What I'm aiming to do is to simplify as much as possible: since the modifications to GPL-2 are in two very specific areas: "Non-GPL Components in Plugins and Drivers" and "Microsoft Distributable Code", then all code using this header will not be in either of those two areas and will therefore be GPL2.

Reply:
If this assumption is incorrect, please let me know @XLTechie